katcp/test/test_resource_client.py

```diff
@@ -744,34 +768,43 @@ def side_effect(*args, **kwargs):
         strat1 = ('period', '2.1')
         strat2 = ('event',)
         strat3 = ('event-rate', '2', '3')
-        yield DUT.set_sampling_strategy('resource1', 'sensor_1', strat1)
+        yield DUT.set_sampling_strategy('resource1_sensor_1', strat1)
         DUT.children.resource1.set_sampling_strategy.assert_called_once_with(
             'sensor_1', strat1)
         DUT.children.resource2.set_sampling_strategy.assert_not_called()
         DUT.children.resource3.set_sampling_strategy.assert_not_called()
         DUT.children.resource1.set_sampling_strategy.reset_mock()
 
-        yield DUT.set_sampling_strategy('resource2','sensor_1', strat2)
+        yield DUT.set_sampling_strategy('resource2.sensor_1', strat2)
```

Comment:
Yes intentional - I wanted to do one with a dot and one with an underscore to check that both work.